internal/customtypes/export_services.go

```diff
@@ -54,9 +57,23 @@ func (es *ExportServices) Set(services string) error {
 		}
 	}
 
-	slices.Sort(serviceList)
+	slices.Sort(returnServiceList)
 
-	*es = ExportServices(serviceList)
+	*es = ExportServices(returnServiceList)
+	return nil
+}
+
+func (es *ExportServices) GetServicesInGroup(serviceGroup string) error {
```

Comment:
Since this is an in-place modification of `es`, this function is better named SetServicesByServiceGroup.

Also if possible, it would be good to have the parameter of type `ExportServiceGroup` instead of `string`